switch.py

```diff
@@ -0,0 +1,100 @@
+import logging
+from .melview import MelViewAuthentication, MelView
+from homeassistant.components.switch import SwitchEntity
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
I think you'll need to add the following.
```python
DOMAIN = 'melview'
REQUIREMENTS = ['requests']
DEPENDENCIES = []
```

Reply:
hmm are you sure? I don't use `requests` here and dont have deps so I don't think these are needed? what does `DOMAIN = 'melview'` do?

Reply:
The `requests` requirement is a requirement of the `melview.py` file, which isn't distributed with its own requirements so it's best to define them here.

The domain setting just tells HA that this is part of the `melview` platform (I think).

Reply:
hmm strange that we have to type these here again. but yeah added anyway